### ISSUE TITLE
Two wheels on bicycle suddenly wasn't enough after autosave.

### ISSUE THREAD

**Issue by woutershep** [CleverRaven/Cataclysm-DDA#17584]
I was driving my newly found bicycle home over the road and then an autosave happened and suddenly i was unable to drive it further. I had to get off and get back on and then i could continue on my way.

Relevant messages:

"Moving onto this extra light frame is slow!",
"There are vehicle controls here.",
"Press ^ to drive.",
"You take control of the Bicycle.",

*happily cycling halfway home here*

"Saving game, this may take a while",
"The Bicycle doesn't have enough wheels to move! x11",

The 10" steerable front wheel was yellow and the 10" unsteerable back wheel was in full health.
It does not seem to happen after every autosave. At least i haven't managed to reproduce it in my brief continuation, so there is probably some complexity in the requirements to trigger this.

**Comment by Zireael07** [comment]
I just ran into the same issue in build 16580 (g188d33f). In my case, both wheels are green. No autosaves in the meantime - I got in, tried to turn the bike around, midway in bam, not enough wheels to move.

Reproduced twice.

How do I get a save to you?

**Comment by Coolthulhu** [comment]
> How do I get a save to you?

**Comment by Coolthulhu** [comment]
Next time it happens try turning debug mode and see where the "crosshair" is. It should be centered in the middle of the vehicle.

**Comment by tivec** [comment]
@Coolthulhu: I am experiencing this with a car that I added an extra engine to (electric).

https://www.dropbox.com/s/11j4af1lanrceek/Fragaria.7z?dl=0

It is the vehicle I am currently at. Just switch to the electric engine, and drive around a bit making some turns. This does not seem to happen on the normal petrol engine in the vehicle.

I turned on debug mode, and this is what I see:
![Screenshot](http://i.imgur.com/xXRYhER.png)
